ci-operator/step-registry/ipi/conf/azure/oidc-creds-provision/ipi-conf-azure-oidc-creds-provision-commands.sh

```diff
@@ -11,9 +11,17 @@ CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
 REGION="${LEASED_RESOURCE}"
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
+if [ ! -f "${CONFIG}" ]; then
+    echo "File not found: ${CONFIG}"
+    exit 1
+fi
 # yq-go is not available in the ci image...
 #BASE_DOMAIN_RESOURCE_GROUP_NAME=$(yq-go r "${CONFIG}" 'platform.azure.baseDomainResourceGroupName')
 BASE_DOMAIN_RESOURCE_GROUP_NAME=$(fgrep 'baseDomainResourceGroupName:' ${CONFIG} | cut -d ":" -f2 | tr -d " ")
+if [ -z "${BASE_DOMAIN_RESOURCE_GROUP_NAME}" ]; then
+    echo "BASE_DOMAIN_RESOURCE_GROUP_NAME is: ${BASE_DOMAIN_RESOURCE_GROUP_NAME}"
```

Comment:
```suggestion
    echo "baseDomainResourceGroupName in install-config.yaml not found"
```

Reply:
Thank you.  In ipi-conf-azure-oidc-creds-provision, this reference causes an exit with code 1, but there are no other logs. I want to print some logs for debugging. ^^